src/current/_includes/metrics-usage/sql.txn.latency.md

```diff
@@ -0,0 +1 @@
+Over the last minute, this cluster executed 90% or 99% of transactions within this time. This time does not include network latency between the cluster and client. These metrics provide an overview of the current SQL workload.
```

Comment:
mention of 'transactions' should be a link to transaction docs as elsewhere

Reply:
added links